app/importers/activity_insight/get_user_data.rb

```diff
@@ -34,7 +34,11 @@ def populate_active_users(users_hashed)
           l_name:    row['Last Name'],
           m_name:    row['Middle Name'],
           college:   row['College'],
-          campus:   row['Campus']})
+          campus:    row['Campus']})
```

Comment:
Now that I think about it some more, maybe it does make sense to set `com_id: nil` here.  That way we guarantee that only users with a college of 'MD' can have a `com_id`.  This would probably negate the need to filter the xml build on 'MD', but we might want to keep that there just to be safe.

Reply:
[757fde8](https://github.com/psu-libraries/fams_tools/pull/46/commits/757fde86d5349ab908daf94a0857596acaf145e9)